firelyserver/releasenotes/roadmap.rst

```diff
@@ -7,52 +7,64 @@ This page lists the features and changes we have planned for the foreseeable fut
 
 Disclaimer: No rights can be derived from this roadmap.
 
-2020
+2021
 ----
 
+Q1
+^^
+
+* Full ONC/CMS compliance
+
+* Firely Server: ONC Edition
+
+   * Downloadable version of Firely Server that is fully compliant with the ONC Cures Act Final Rule
+
+* CAR factory/files
+
+   * Greatly reduces start-up time of Firely Server and other services
+
+
+* Firely Validator (CLI)
+
+   * Separate validation service as a CLI tool
+
+
 Q2
 ^^
 
-* FHIR Mapping improvements to support logical models like CDA
-* Better support for Sorting, starting with dates and times
-* Integration with external Terminology services
-* Patch
-* Audit log & Audit events
+* Bulk data export for MongoDB
+
+* Bulk data import tool for MS SQL Server
+
+   * CLI tool for bulk ingestion of data into Firely Server
+
+* Firely Validator for cloud environments
+
+   * Separate validation service optimized for usage in cloud environments
 
-* R4 search features
+* Firely Server: CMS Edition
 
-   * Search on reference by identifier
+   * Downloadable version of Firely Server that is fully compliant with the CMS Final Rule
+
+* Full compliance with MedMij standard
```

Comment:
Given the first item in Q3 I think you meant to remove it here.

Reply:
No, it are two different things: the one is actually being compliant and the other is offering a download with all the IGs preloaded as a solution, like we will for the ONC soon.